public/New-DbaDbUser.ps1

```diff
@@ -69,165 +74,246 @@ function New-DbaDbUser {
     .EXAMPLE
         PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Login user1
 
-        Creates a new sql user with login named user1 in the specified database.
+        Creates a new sql user named user1 for the login user1 in the database DB1.
 
     .EXAMPLE
-        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username user1
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -User user1
 
-        Creates a new sql user without login named user1 in the specified database.
+        Creates a new sql user named user1 without login in the database DB1.
 
     .EXAMPLE
-        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Login Login1 -Username user1
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -User user1 -Login login1
 
-        Creates a new sql user named user1 mapped to Login1 in the specified database.
+        Creates a new sql user named user1 for the login login1 in the database DB1.
 
     .EXAMPLE
-        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Login Login1 -Username user1 -DefaultSchema schema1
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -User user1 -Login Login1 -DefaultSchema schema1
 
-        Creates a new sql user named user1 mapped to Login1 in the specified database and specifies the default schema to be schema1.
+        Creates a new sql user named user1 for the login login1 in the database DB1 and specifies the default schema to be schema1.
 
     .EXAMPLE
-        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username "claudio@********.onmicrosoft.com" -ExternalProvider
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -User "claudio@********.onmicrosoft.com" -ExternalProvider
 
-        Creates a new sql user named 'claudio@********.onmicrosoft.com' mapped to Azure Active Directory (AAD) in the specified database.
+        Creates a new sql user named 'claudio@********.onmicrosoft.com' mapped to Azure Active Directory (AAD) in the database DB1.
 
     .EXAMPLE
-        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username user1 -Password (ConvertTo-SecureString -String "DBATools" -AsPlainText)
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username user1 -Password (ConvertTo-SecureString -String "DBATools" -AsPlainText -Force)
 
-        Creates a new cointained sql user named user1 in the database given database with the password specified.
+        Creates a new contained sql user named user1 in the database DB1 with the password specified.
     #>
+
+    ### This command has more comments than other commands, because it should act as an example for other commands.
+    ### These extra lines start with "###" and should help new contributores to understand why we code the way we do.
```

Comment:
"new contributors" instead of "contributores"

Reply:
Thanks.